rpm/podman.spec

```diff
@@ -93,6 +93,7 @@ BuildRequires: systemd
 BuildRequires: systemd-devel
 Requires: catatonit
 Requires: conmon >= 2:2.1.7-2
+Requires: virtiofsd
```

Comment:
Hmm, I thought we talked about a new `podman-machine` package where this requirement could live (and we could add qemu there, or even in the future libvirt say)?

It's definitely not required to just run containers right now.

Reply:
Also, as long as we're touching the spec, we also must add a `ln -sr /usr/libexec/virtiofs /usr/libexec/podman/` into it...which I think would be best done in that new "mostly metadata" subpackage.

Reply:
im not going to wait for that.

Reply:
Hmm...but are you *sure* you want to pull in virtiofsd everywhere today? It seems inconsistent as it's only a requirement for `podman machine`, but we don't also depend on qemu.

Reply:
IMO I don't think a requires on the podman package is right, it is not required for normal operation on linux, podman machine is some extra most users don't need which why I suggested creating a podman-machine package which holds the deps.

Reply:
Also just doing virtiofsd is inconsistent as qemu is not required.

Reply:
@baude subpackage for podman-machine which only fetches the additional deps will probably cause the least pushback from other projects. machine users will explicitly need to `dnf install podman-machine` then.

Reply:
Set it up as recommends then or suggests.  I don't think it is a big ideal if the package does not exists.  But the podman machine commands will fail if it is not their by mistake.  This means podman machine needs to be made smarter when qemu is not available.

Reply:
Actually, because we look in $PATH for qemu, and after some debate due to the variations of virtiofsd's location on different OS/distros we ended up *requiring* a `/usr/libexec/podman` symlink...the status quo will just break podman machine users on these platforms unless they add that symlink into their packaging.

Reply:
@baude i have a draft PR at https://github.com/containers/podman/pull/23095 

@cgwalters got a link to the symlink change please?